website/src/documentation/io/built-in-google-bigquery.md

```diff
@@ -262,13 +265,43 @@ in the following example:
 {% github_sample /apache/beam/blob/master/sdks/python/apache_beam/examples/snippets/snippets.py tag:model_bigqueryio_read_query_std_sql
 %}```
 
+### Using the BigQuery Storage API {#storage-api}
+
+The [BigQuery Storage API](https://cloud.google.com/bigquery/docs/reference/storage/)
+allows you to directly access tables in BigQuery storage. As a result, your
+pipeline can read from BigQuery storage faster than previously possible.
+The Beam SDK for Java (version 2.11.0 and later) supports using the
+[BigQuery Storage API](#storage-api) when reading from a table. Using the
+BigQuery Storage API with a query string is not supported.
+
+***Note:*** The SDK for Python does not support the BigQuery Storage API.
+
+The BigQuery Storage API is distinct from the existing BigQuery API. You must
+[enable the BigQuery Storage API](https://cloud.google.com/bigquery/docs/reference/storage/#enabling_the_api)
+for your Google Cloud Platform project. Then, add
+`.withMethod(Method.DIRECT_READ)` to your pipeline code when you read from a
+BigQuery table. The following example uses `read(SerializableFunction)` and the
+BigQuery Storage API.
+
+```java
+PCollection<KV<String, Long>> output =
+    p.apply(BigQueryIO.read(new ParseKeyValue())
+                      .from(tableSpec)
+                      .withMethod(Method.DIRECT_READ);
```

Comment:
This covers how to use the new API, but in order to use some new features -- specifically column selection and filter push-down -- callers must also specify a TableReadOptions proto [1] using the .withReadOptions() method.

[1] https://github.com/googleapis/google-cloud-java/blob/c011130c0114ccdcde9bda60202f334722df331d/google-api-grpc/proto-google-cloud-bigquerystorage-v1beta1/src/main/proto/google/cloud/bigquery/storage/v1beta1/read_options.proto

Reply:
Added